src/sentry/static/sentry/app/views/settings/projectAlerts/projectAlertRules.jsx

```diff
@@ -212,6 +226,7 @@ class ProjectAlertRules extends AsyncView {
               params={this.props.params}
               routes={this.props.routes}
               onDelete={this.handleDeleteRule.bind(this, rule)}
+              firstRule={this.state.ruleList.indexOf(rule) === 0}
```

Comment:
More idiomatic to use the indexed form of `map` to get the index. See https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/stream.jsx#L50

Reply:
done